### DISCUSSION TITLE
Anycubic Photon Mono M3 Plus / M3 Max - exposure time finder

### DISCUSSION THREAD

**Discussion by AndiK666** [sn4k3/UVtools#491] [Q&A]
For days I've been trying to create a test file for my M3 Plus and M3 Max with UVtools using the exposure time finder.
Unfortunately unsuccessful so far.
So I was able to create the files, but the printer makes each exposure on a new level. However, UVtools provides that the test fields that are on the same level should be exposed individually. In the Anycubic RERF test, all test fields are exposed together and the fields go out one after the other. Then the next level is exposed according to the same scheme.
In the files I create, the base levels are exposed together and then each test object individually, one after the other. The printer goes one level higher every time, which it shouldn't do... Accordingly, I've failed with my exposure attempts so far.
Does anyone of you have experience with exposure time finder and the Anycubic Photon Mono M3 Plus / M3 Max?
It is such a powerful and helpful tool that I would like to understand.

**So far I have followed the following instructions:**
https://www.youtube.com/watch?v=lRbZw6GbGCI&t
https://www.youtube.com/watch?v=vf_2OZgDCMM

**These are the exposure time finder files I created:**
[M3 Plus 1.2s-3.3s (0.10x21) - Elegoo ABS Like Rot.zip](https://github.com/sn4k3/UVtools/files/8855509/M3.Plus.1.2s-3.3s.0.10x21.-.Elegoo.ABS.Like.Rot.zip)
[M3 Max 1.2s-6.0s(0.15x33) - Elegoo ABS Like Rot.zip](https://github.com/sn4k3/UVtools/files/8855510/M3.Max.1.2s-6.0s.0.15x33.-.Elegoo.ABS.Like.Rot.zip)

**This is what the result looks like when I open the files with Anycubic Photon Workshop 64:**
-Unfortunately, the printer does exactly what the preview shows.
-Thus, the printing plate is soon floating in the air and the exposures only take place on the FEP without contact to the plate.

![2022-06-07 17_34_47-Anycubic Photon Workshop V2 1 30 RC17](https://user-images.githubusercontent.com/107070098/172451526-0a6e6309-8701-4a8e-8cbb-6e87b75d7a80.jpg)

**Comment by sn4k3** [comment]
This is a very asked problem.
See here: https://github.com/sn4k3/UVtools/discussions/489
